src/aws/flb_aws_util.c

```diff
@@ -581,6 +581,8 @@ void flb_aws_print_error(char *response, size_t response_len,
 
     error = flb_json_get_val(response, response_len, "__type");
     if (!error) {
+        /* error can not be parsed, print raw response */
+        flb_plg_warn(ins, "Raw response: %s", response);
```

Comment:
This lacks full context that it could have: we can print the `api` value in the raw response here, like: `PutLogEvents: Raw Response: `

Reply:
I guess based on the response, the API should be obvious? Is this true? I think if this does really add context, it'd be nice to add. 

We could also follow up with it later I suppose, as long we stage it now, so we don't forget it..

What are your thoughts on how important this context is?

Reply:
I agree this is nice to have. I'll update this for master, and include it in aws-for-fluent-bit after release of 2.32.0

Reply:
Cool and thanks!

Reply:
Here: https://github.com/fluent/fluent-bit/pull/8122/files